GPSTest/src/main/java/com/android/gpstest/GpsTestActivity.java

```diff
@@ -983,7 +983,7 @@ private synchronized void gpsStart() {
             if (minTime != (long) (Double.valueOf(getString(R.string.pref_gps_min_time_default_sec))
```

Comment:
This can still be changed to `Double.parseDouble()` to match the other improvements.

Reply:
Ugg! I should have caught that.. Thanks for the sharp eyes! 8-)

It should be fixed now..